code/modules/pda/pdas.dm

```diff
@@ -90,6 +90,8 @@
 	icon_state = "pda-h"
 
 /obj/item/pda/heads/ert
+	default_cartridge = /obj/item/cartridge/centcom
+	detonate = FALSE
```

Comment:
While you're here, mind changing the definition in `/obj/item/pda` to `var/detonate = TRUE`, and the (only other) override in `/obj/item/pda/silicon` to `detonate = FALSE` ?